### PR TITLE
Release Notes August 2022

### PR DESCRIPTION
**Pull request by can-memfault**

### CODE REVIEW

**Comment by can-memfault** [comment]
@HBehrens do you mind taking a last look? I checked the links but it was pretty late, so another set of eyes would be appreciated :)

File: blog/2022-08-31.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  and
+  [hiding Issues with no Traces](/docs/platform/issue-management/#automated-hiding-of-issues-with-zero-traces)
+- Added: Panels under Device Timeline to show relevant metrics together for
+  Android and Linux projects
+- Improved:
+  [Searching for Devices](https://app.memfault.com/organizations/-/projects/-/devices)
+  via [Timeseries Metrics](/docs/platform/metrics/#timeseries) and
+  [Attributes](/docs/platform/metrics/#attributes) now case insensitive
+- Improved: UI feedback when accessing a deleted Trace
+- Improved: [Memfault Docs](/) search shows Web App results above Changelog
```
Comment (HBehrens) [inline blog/2022-08-31.mdx:55]: I should have caught that earlier, sorry
```suggestion
- Improved: [Memfault Docs search](/search/) shows Web App results above Changelog
```

File: blog/2022-08-31.mdx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    for [CAT1A (PSoC™️ 6)](https://github.com/Infineon/mtb-pdl-cat1)
+  - Added:
+    [New example application](https://github.com/memfault/memfault-firmware-sdk/blob/master/examples/nrf-connect-sdk/nrf5)
+    for trying Memfault on nRF53 & nRF52
+  - Added: Option to print Zephyr backtrace output to console while also using
+    Memfault
+  - Improved: Support for upcoming Zephyr 3.2.0 (and also 3.1.0) with the newly
+    namespaced Zephyr include path
+  - Fixed:
+    [Heap stats free not checking in_use flag](https://github.com/memfault/memfault-firmware-sdk/issues/32)
```
Comment (HBehrens) [inline blog/2022-08-31.mdx:110]: maybe a shoutout to @christophgreen as [Noah did in the comments](https://github.com/memfault/memfault-firmware-sdk/issues/32#issuecomment-1219487598) – always nice for people to receive recognition

File: blog/2022-08-31.mdx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+- Added: [Metrics](/docs/linux/linux-metrics) incl. grouping via Device Timeline
+- Added: Fleet-wide metric aggregations (sum, count, min, max, mean)
+- Improved: [Getting Started guide](/docs/linux/linux-getting-started-guide) now
+  shown after project creation
+- Improved: Documentation about
+  [Linux OTA](/docs/linux/linux-releases-integration-guide)
+- SDK version
+  [0.3.0](https://github.com/memfault/memfault-linux-sdk/blob/-/CHANGELOG.md#030---2022-08-31)
+  was released. Some highlights:
```
Comment (HBehrens) [inline blog/2022-08-31.mdx:139]: Yup, precisely on Aug 31st , the changelog states. Nicely done, team :)

**Review by HBehrens — APPROVED** [review]
LGTM, a few minor nits

File: blog/2022-08-31.mdx (hunk 1 above)
Comment (can-memfault) [reply blog/2022-08-31.mdx:55]: Uu nice, I didn't know we can link to search :)

**Pull request merged**